lab04_example/include/lab04_example/VelocityController.h

```diff
@@ -0,0 +1,73 @@
+/* Author: Luke Dennis l.dennis@unsw.edu.au */
+
+#include <ros/ros.h>
+#include <std_msgs/Int8.h>
+#include <geometry_msgs/Twist.h>
+#include <lab04_example/key_control.h>
+#include <lab04_example/KeyToCmdVel.h>
+
+namespace Lab04 {
+    /* Inspiration from:
+     *  https://github.com/wsnewman/ros_class/blob/master/example_ros_class/src/example_ros_class.h
+     *  https://roboticsbackend.com/roscpp-timer-with-ros-publish-data-at-a-fixed-rate/
+     */
+    class VelocityController {
+        public:
+            VelocityController(ros::NodeHandle nh) : nh_(nh) {
+                key_control_sub_ = nh_.subscribe("KeyControl", 10,
```

Comment:
Recommend some commentary around variable style here, or links to resources discussing style.

Reply:
I think this is outside the scope but here are some (dense) resources:
https://google.github.io/styleguide/cppguide.html
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines